### PR TITLE
feat: 适配「词祭」界面主题

### PR DESCRIPTION
**Pull request by SherkeyXD**

### CODE REVIEW

**Comment by Constrat** [comment]
I'm guessing no solution currently for Mall task since it's full OCR. And OCRing this is kinda hard, triangles, squares lol
```json
    "Mall": {
        "algorithm": "OcrDetect",
        "action": "ClickSelf",
        "ocrReplace": [["米购", "采购"]],
        "text": ["采购", "购中", "中心", "到期"],
        "roi": [740, 440, 200, 100],
        "next": ["CreditStoreOcr", "Mall", "Mall@CloseAnnos#next"],
        "postDelay": 500
    },
```

Does this work? 
image

**Comment by SherkeyXD** [comment]
> I'm guessing no solution currently for Mall task since it's full OCR. And OCRing this is kinda hard, triangles, squares lol
> ```json
>     "Mall": {
>         "algorithm": "OcrDetect",
>         "action": "ClickSelf",
>         "ocrReplace": [["米购", "采购"]],
>         "text": ["采购", "购中", "中心", "到期"],
>         "roi": [740, 440, 200, 100],
>         "next": ["CreditStoreOcr", "Mall", "Mall@CloseAnnos#next"],
>         "postDelay": 500
>     },
> ```
> 
> Does this work? 
> image
> 

Maybe we should shift itlol template lol

**Comment by Constrat** [comment]
Yeah... We survived the Sami one, but this one introduces too much garbage. You can check if the task works by just entering the interface selection, since Credit Store just detects it (or just DebugTask lol)

**Comment by SherkeyXD** [comment]
Done shifting

**Comment by Manicsteiner** [comment]
这样的采购中心template显然是不行的，上半部分都可能被挡住 #10194

**Comment by SherkeyXD** [comment]
草 那我一会放右边去

**Comment by Constrat** [comment]
Wait before merge, so we add Global templates, as modifying the main tasks.json will break all other clients

File: resource/tasks.json [hunk 1]
```diff
@@ -3161,16 +3185,17 @@
         "next": ["Mall@ReturnButtons#next", "Mall", "Mall@CloseAnnos#next"]
     },
     "Mall": {
-        "algorithm": "OcrDetect",
+        "template": ["Mall.png", "MallDark.png", "MallSami.png", "MallMistCity.png", "MallSiege.png", "MallSarkaz.png"],
```
Comment (Constrat) [inline resource/tasks.json:3188]:
```suggestion
        "template": ["MallOrigin.png", "MallDark.png", "MallSami.png", "MallMistCity.png", "MallSiege.png", "MallSarkaz.png"],
```

**Review by Constrat — CHANGES_REQUESTED** [review]

File: resource/tasks.json (hunk 1 above)
Comment (Constrat) [reply resource/tasks.json:3188]: Unless you want to change the template name, which is probably better as that would follow the format we've been using already.

File: resource/tasks.json [hunk 2]
```diff
@@ -3161,16 +3185,17 @@
...
         "next": ["CreditStoreOcr", "Mall", "Mall@CloseAnnos#next"],
-        "postDelay": 500
+        "postDelay": 500,
+        "crop_doc": {
+            "roi": [822, 457, 86, 71],
+            "mask": []
+        }
     },
     "CreditStoreOcr": {
-        "algorithm": "OcrDetect",
```
Comment (Constrat) [inline resource/tasks.json:3173]: Oh, this is the issue!

**Review by Constrat — CHANGES_REQUESTED** [review]

**Comment by Constrat** [comment]
Let's just change the template name to keep consistency:
MallOrigin.png -> Mall.png

File: resource/tasks.json (hunk 2 above)
Comment (Constrat) [reply resource/tasks.json:3173]: Fixed with 
1f37351

**Comment by SherkeyXD** [comment]
I misnamed it lol
Thanks for the fix

**Comment by Constrat** [comment]
Still failing 
image

**Comment by Constrat** [comment]
By the way, there's no rush in merging. The theme was just preloaded so we have a lot of time to implement both `KR` and `txwy`

**Comment by Manicsteiner** [comment]
> Should come out in September.

And Rainbow Six Siege theme will comes to Yostar servers soon on September 5th

**Comment by Constrat** [comment]
Oh yeah, that's true. But that might be extending it a bit too much. We can do that separately since no breaking changes are happening on the CN with that.

**Comment by SherkeyXD** [comment]
Is it ready?

**Comment by Constrat** [comment]
> Is it ready?

I think txwy misses the templates :P

**Review by ABA2396 — APPROVED** [review]

**Pull request merged**